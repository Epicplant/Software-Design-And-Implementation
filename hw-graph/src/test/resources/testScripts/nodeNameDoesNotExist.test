# Tests adding an edge between a node that exists and a node that doesn't

# Creates a new graph
CreateGraph newGraph

# Creates the edges' first node
AddNode newGraph n1

# Creates an edge from n1 to n2 with weight 2 that shouldn't exist
AddEdge newGraph n1 n2 2

# Lists only n1
ListChildren newGraph n1