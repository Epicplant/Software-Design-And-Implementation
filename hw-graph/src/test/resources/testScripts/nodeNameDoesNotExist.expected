# Tests adding an edge between a node that exists and a node that doesn't

# Creates a new graph
created graph newGraph

# Creates the edges' first node
added node n1 to newGraph

# Creates an edge from n1 to n2 with weight 2 that shouldn't exist
added edge 2 from n1 to n2 in newGraph

# Lists only n1
the children of n1 in newGraph are: